lib/enter.js

```diff
@@ -16,7 +16,7 @@ module.exports = function (tree) {
     if (transitions) {
       enter.classed('transitioning-node incoming-node', true)
       tree._forceRedraw() // Force a redraw so we see the updates transitioning
-      enter.classed('transitioning-node incoming-node', false)
+      enter.classed('transitioning-node', false)
     }
```

Comment:
Actually, now that I'm looking at this again after a break from it. That entire class isn't even necessary. We can just apply a default transition to the transitioning node. It doesn't need an opacity set because the default is 1. Updated.